src/fty_nut_command_server.cc

```diff
@@ -207,7 +207,7 @@ treat_pending_commands(nut::Client &nutClient, mlm_client_t *client, PendingComm
                 try {
                     switch (nutClient.getTrackingResult(id)) {
                         case nut::FAILURE:
-                            command.errMsg = "Failure to execute command";
+                            command.errMsg = "Failed to execute command on power device.";
```

Comment:
I was expecting a TRANSLATE_ME() here.
@EldoreiJK could you please confirm on the actual approach to have for this PR? thx a lot

Reply:
If this message makes it to the UI, then yes, otherwise no need.

Reply:
thx @EldoreiJK